portal/forms/home.py

```diff
@@ -63,11 +63,9 @@ def clean_name(self):
 
     def clean_message(self):
         message = self.cleaned_data.get("message", None)
```

Comment:
Does this not return an already encoded string by django?

Reply:
I think it should be, but for some reason it isn't encoding properly. This was talked about in https://github.com/ocadotechnology/codeforlife-portal/pull/578, this is probably still a temporary fix but it is better than restring the input to ascii characters only.